src/test/java/net/snowflake/client/log/JDK14JCLWrapperLatestIT.java

```diff
@@ -63,10 +63,12 @@ public void setUp() {
     // Set debug level to lowest so that all possible messages can be sent.
     logger.setLevel(Level.FINEST);
     logger.addHandler(this.handler);
+    logger.setUseParentHandlers(false);
   }
 
   @After
   public void tearDown() {
+    logger.setUseParentHandlers(true);
```

Comment:
Add parent log handlers back in at the end to reset to how things were prior to the test.